ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/main/java/de/adorsys/ledgers/baam/db/domain/TellerAccess.java

```diff
@@ -0,0 +1,42 @@
+package de.adorsys.ledgers.baam.db.domain;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import jakarta.persistence.*;
+import java.util.Date;
+import java.util.UUID;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class TellerAccess {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    private String name;
+    private String tellerId;
+
+
+    @Enumerated(EnumType.STRING)
+    private AccessStatus status;
+
+    private double dailyLimit;
+
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date createdDate;
+
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date lastModifiedDate;
+
+
+    private String accessType = "TELLER_ACCESS";
+}
```

Comment:
Hello  @yvanhenang

To make these clear, i think you should extend your class to the BankAccountAcess entity.

let's use inheritance as the BankAccountAcess entity serves as an entery point to all accesses